lib/base-command.js

```diff
@@ -20,6 +20,10 @@ class BaseCommand {
     return this.constructor.description
   }
 
+  get ignoreImplicitWorkspace () {
+    return !!this.constructor.ignoreImplicitWorkspace
+  }
```

Comment:
why not just `static ignoreImplicitWorkspace = false`? subclasses can override it the same.

Reply:
This follows the pattern we have for the other static attributes.  The `!!` isn't really needed. But we need to add to the tests that check that every command has a name, et al defined so that this attribute is on that list.

Reply:
sure - but how would that not be the case with a static field on the base command?

Reply:
We don't currently define any static fields on the base command.  The pattern today is that every command has to define them, and we enforce that constraint in tests.  It's our current decision for balancing intentionality and dry/terseness.

Reply:
The pattern right now has no static properties in the base command. Following the pattern of other static properties, added it as a required property in the implementation test, removed the `!!`, and updated all of the commands.

Reply:
I'm pretty confused; i must just not understand what a "base command" is.